app/views/finders/_select_facet.html.erb

```diff
@@ -4,7 +4,7 @@
   :title => select_facet.name,
   :aria_controls_id => "js-search-results-info",
   :options_container_id => select_facet.key,
-  :options => select_facet.options,
+  :options => select_facet.options("js-search-results-info", select_facet.key),
```

Comment:
It looks like we could avoid options requiring params. Is this the only place `select_facet.options` is used?

Reply:
@bilbof not quite sure what you're suggesting but agree that this probably isn't the cleanest way of doing it. Let's catch up and discuss it in person.

Reply:
@sihugh Bill's on leave now and I'm not sure I understand this comment. If you have time could you look at this PR and see if it's okay?